moveit_commander/test/python_moveit_commander_ros_namespace.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2012, Willow Garage, Inc.
```

Comment:
Please update the copyright. This is either you, your company (if you work on this for your job).
If you own the copyright, you can also pass it to `Open Source Robotics Foundation`.